sktime/forecasting/ensemble_algorithms.py

```diff
@@ -0,0 +1,234 @@
+from sklearn.base import BaseEstimator
+from scipy.optimize import nnls
+import numpy as np
+
+
+class EnsembleAlgorithm(BaseEstimator):
+    """Wrapper class to take ensemble algorithms and allow them to be easily set
+
+    Parameters
+    ----------
+    n : float
+    loss_func : function
+    """
+    def __init__(self, n=10, loss_func=None, **kwargs):
```

Comment:
no need for `**kwargs`, right? Here and below

Reply:
Yes you are right